### PR TITLE
About this Release page

### PR DESCRIPTION
**Pull request by samgnaniah**
Updates the About this Release page and adds some initial content for the Design section.

### CODE REVIEW

File: en/docs/get-started/quick-start-guide/integration-qsg.md [hunk 1]
```diff
@@ -224,4 +224,4 @@ Here you will not see an onboarding page but a listing of the deployed services
 ## What's next?
 
 - [Develop your first integration solution]({{base_path}}/integrate/develop/integration-development-kickstart).
-- Try out the **tutorials** and **examples** available in the [Learn section of our documentation]({{base_path}}/integrate/learn-overview/#integration-use-cases).
+- Try out the **tutorials** and **examples** available in the [Integrate section of our documentation]({{base_path}}/integrate/learn-overview/#integration-use-cases).
```
Comment (nilminiwso2) [inline en/docs/get-started/quick-start-guide/integration-qsg.md:227]: Tutorials are not in the 'Integrate' section

File: en/docs/get-started/about-this-release.md [hunk 2]
```diff
@@ -1,63 +1,229 @@
...
 
-WSO2 API Manager comprises of the following modules:
+For more information on WSO2 API Manager, see the [overview in the WSO2 API Manager 4.0.0 documentation]({{base_path}}/getting-started/overview/).
 
-- **API Publisher**: Allows API creators to design, implement, and document APIs and allows API product managers to manage the lifecycle of APIs and create API products by using one or more APIs.
+## What is new in this release
 
-- **Developer Portal**: A portal for onboarding application creators. It allows application creators to discover, subscribe, test and consume APIs through their applications.
+The **WSO2 API Manager 4.0.0-Alpha** is the **latest** **WSO2 API Manager release** and is the successor of **WSO2 API Manager 3.2.0**. It contains the following components, which have new features and enhancements:
```
Comment (nilminiwso2) [inline en/docs/get-started/about-this-release.md:9]: shouldn't we say this is the successor of EI as well?

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/get-started/quick-start-guide/integration-qsg.md:227]: The tutorials list is available here right? We don't have a list of them anywhere else.

File: en/docs/get-started/about-this-release.md [hunk 3]
```diff
@@ -1,63 +1,229 @@
...
-## Known issues
+See [details of all the changes including improvements, and bug fixes in this release](https://github.com/wso2/product-apim-tooling/issues?q=is%3Aissue+is%3Aclosed+label%3A4.0.0).
+
+#### Known Issues
+
+All the open issues pertaining to WSO2 API Manager-4.0.0-Alpha are reported [here](https://github.com/wso2/product-apim-tooling/issues?q=is%3Aopen+is%3Aissue).
+
+---
+
+### Integration Studio
```
Comment (nilminiwso2) [inline en/docs/get-started/about-this-release.md:163]:
```suggestion
### WSO2 Integration Studio
```

File: en/docs/get-started/about-this-release.md (hunk 2 above)
Comment (samgnaniah) [reply en/docs/get-started/about-this-release.md:9]: Will check on whether we should say that.

File: en/docs/get-started/quick-start-guide/integration-qsg.md (hunk 1 above)
Comment (samgnaniah) [reply en/docs/get-started/quick-start-guide/integration-qsg.md]: For now I will remove mention of tutorials until the scenario tutorials come in.

**Review by RukshiW — APPROVED** [review]

**Pull request merged**